index.html

```diff
@@ -107,18 +107,68 @@ <h3>Overview</h3>
 					packaging. And as web-compatible file sets, they are adaptable to future changes to publishing
 					formats.</p>
 			</section>
-
-			<section id="conformance"></section>
-
+			
+			<section id="relationship-epub3" class="informative">
+				<h3>Relationship to EPUB 3</h3>
+				
+				<p>The [=eBraille file set=] is designed to be compatible with EPUB 3 [[epub-33]] and adapt to changes
+					to that standard. Distribution and consumption on mainstream [=EPUB reading systems=], however, is
+					not a primary goal of this format. This specification introduces some additional requirements and
+					features beyond those defined in EPUB 3, and it is not expected that mainstream reading systems will
+					adapt to these modifications. Consequently, an eBraille publication may not render exactly as
+					intended outside of eBraille reading systems.</p>
+				
+				<p>The primary difference between the eBraille format and EPUB 3 is eBraille publications do not have to
+					be packaged and distributed in an [=EPUB container=] [[epub-33]]. An [=eBraille publication=] will
+					be easy to package in an EPUB container when needed, however.</p>
+				
+				<p>Compatibility with EPUB 3 also means that eBraille relies on the same <a
+					data-cite="epub-33#sec-intro-relations">underlying technologies</a> [[epub-33]]. Many of these
+					technologies, like [[html]], are referred to as "evergreen" or "living" standards as they are
+					updated often and they do not have version numbers. Others, like [[svg]], are referred to without a
+					specific version number so that the latest recommendation is always the recommended one to use.</p>
+				
+				<p>The practical effect of this approach is predominantly beneficial for [=eBraille creators=]. New
+					features become available for use as soon as they are standardized, without having to wait for the
+					eBraille standard to catch up. It does come with some drawbacks, however. eBraille creators also
+					have to keep themselves apprised of changes to the underlying technologies, and use their best
+					judgement in some cases as to whether a new feature is supported well enough in [=eBraille reading
+					systems=] to begin deploying. In some rare cases, it may also mean that content that was previously
+					valid may no longer be, but breaking changes like this usually only happen when features are
+					unsupported or are found to have serious security or privacy flaws.</p>
+			</section>
+			
+			<section id="future-dir" class="informative">
+				<h3>Future directions</h3>
+				
+				<p>The first version of the eBraille format is focused on creating a reading experience that is
+					minimally feature complete for the majority of braille publications. It is not expected that this
+					version will handle every unique formatting requirement of every publication type, but future drafts
+					and versions of this specification will focus on making the format more feature complete.</p>
```

Comment:
Should we also say here that the focus lies primary on electronic
reading systems (more specifically refreshable braille displays), and
less on embossing tools? I noticed we don't mention embossers at all
yet. I think the use case deserves to be mentioned, but we should make
it clear that it is not the primary focus in the first version.

I think I would also like to clarify somewhere that:

1. It is not expected that authors always provide complete CSS style
   sheets. If a publication does not have any style sheets embedded,
   it does not mean it will not be formatted at all, or that it can not be
   formatted correctly. It is expected that reading systems apply some
   default styles that make sense for the current locale.

2. Reading systems may apply styling through other means than CSS.

3. We explain why we don't define any CSS extensions yet. In addition
   we could also explain why we intentionally omit certain standard
   CSS features, especially those that are not well supported in
   browsers, from our guidelines. The reason is that we want to wait
   and see which technologies are chosen by implementors and how they
   approach certain formatting requirements.

   E.g. CSS has a "Paged Media" module, but it is not well supported,
   so it does not make much sense to use this module in our best
   practices if it turns out to be very difficult to support by
   implementors.

   Another reason for keeping the guidelines minimal in the first version
   is that it remains to be seen how the majority of the devices will
   work: will they render the document in a "paged" way (breaking it up
   into discrete pages), or a "scrolling" way (like in a web browser).